README.md

```diff
@@ -35,7 +35,12 @@ ActionController::Base.session_store = :redis_store,
 A brief run-down of these options...
 
 - **servers** is an Array of Redis server URLs that we will attempt to find
-  data from.
+  data from.<br>
+  The "session" at the end of the URL is used as the prefix of the key in the redis server.<br>
```

Comment:
I am sorry for my poor expression. 
I just wanted to explain that "session" is the prefix of the key in the Redis server.

for example, 
```
127.0.0.1:6379[0]> keys *
1) "session:1::adb423fbd1bbffe8f3d1b4925ba3cafb69da51ea5160152795cd1ceda0778af8"
``` 

why because I wanted to add this explanation, 
I had confused what is different ```'redis://localhost:6379/0'``` and ```'redis://localhost:6379/0/session'```. I thought it is just the different URL. "session" doesn't have any meaning. 

so, I'd like to add easier explanation about it.

make sense?

thank you so much!

Reply:
I have changed my explanation.
can you please review it once again?

thank you so much.